arangopipe/arangopipe/arangopipe_storage/arangopipe_api.py

```diff
@@ -243,43 +242,41 @@ def lookup_modelperf(self, tag_value):
         mperf_info = None
         mperf_keys = [doc for doc in cursor]
         if len(mperf_keys) == 0:
-            logger.info(
-                "The model performance for tag: "
-                + tag_value
-                + " was not found in Arangopipe!"
-            )
+            logger.info("The model performance for tag: " + tag_value +
+                        " was not found in Arangopipe!")
         else:
             mperf_info = mperf_keys[0]
 
         return mperf_info
 
     def init_graph(self) -> None:
         """Initialize a graph when an instance of ArangoPipe is provisioned."""
-        db_serv_host = self.cfg["arangodb"]["DB_service_host"]
-        db_serv_port = self.cfg["arangodb"]["DB_service_port"]
-        db_name = self.cfg["arangodb"]["dbName"]
-        db_user_name = self.cfg["arangodb"]["username"]
-        db_passwd = self.cfg["arangodb"]["password"]
-        db_conn_protocol = self.cfg["arangodb"][self.mscp.DB_CONN_PROTOCOL]
-
-        host_conn_str = (
-            db_conn_protocol + "://" + db_serv_host + ":" + str(db_serv_port)
-        )
-        client = ArangoClient(hosts=host_conn_str)
-
-        self.db = client.db(
-            name=db_name, username=db_user_name, password=db_passwd, verify=True
-        )
+        # db_serv_host = self.cfg["arangodb"]["DB_service_host"]
+        # db_serv_port = self.cfg["arangodb"]["DB_service_port"]
+        # db_name = self.cfg["arangodb"]["dbName"]
+        # db_user_name = self.cfg["arangodb"]["username"]
+        # db_passwd = self.cfg["arangodb"]["password"]
+        # db_conn_protocol = self.cfg["arangodb"][self.mscp.DB_CONN_PROTOCOL]
+
+        # host_conn_str = (db_conn_protocol + "://" + db_serv_host + ":" +
+        #                  str(db_serv_port))
+        # client = ArangoClient(hosts=host_conn_str)
+
+        # self.db = client.db(name=db_name,
+        #                     username=db_user_name,
+        #                     password=db_passwd,
+        #                     verify=True)
 
         if self.db is None:
             return
```

Comment:
raise an exception (e.g. a ValueError) here now that we expect this as an input argument?

Reply:
Good point. I've also added this check to the constructor.